### PR TITLE
Proposed license and minor refactor/grammar in README

### PR DESCRIPTION
**Pull request by troglobit**
Hi,

thanks for fixing the annoying issue with switcher placement.  Here's a silly pull request with some minor fixes to the `README.md` and a proposed licsene for the project.

Cheers
 /Joachim

### CODE REVIEW

**Comment by berlam** [comment]
Hi Joachim,

thank you for the fixes. I am not sure, if I can make it officially a ISC project, as the original project hasn't got a license yet. That is why I have written _TBD_ under license. The PR for a license in @jorenheit 's project is still open.

Furthermore I would ask you, if you could make a squash commit for the PR as the detailed history for the README and LICENSE is not necessary.

**Comment by troglobit** [comment]
Aha, I hadn't seen your PR at the upstream/original.  Funny we both propose the same license. No problem, understand completely.

About the squash, I'll see what I can do later this week.

**Comment by berlam** [comment]
I assume, that the project from @jorenheit is kind of deprecated or unmaintained as he does not respond to messages or PRs. Which leads me to the idea, that he will be ok with a open-source license. Anyway, the account still looks active to me. 

The ISC license is easy to understand and developer friendly. I use it in almost any open-source project.
Nice to see, that you would put the same license on the project. :+1:

**Comment by berlam** [comment]
PR was merged minutes ago. Thank you @jorenheit !

File: README.md [hunk 1]
```diff
@@ -16,57 +16,61 @@ Features:
...
+    awful.key({ "Mod1", "Shift"   }, "Tab",
+      function ()
+          switcher.switch(-1, "Alt_L", "Tab", "ISO_Left_Tab")
+      end),
 ```
 
 ## Credits ##
 
-This plugin was created by @jorenheit and all the initial credits belong to him.
+This plugin was created by [Joren Heit](https://github.com/jorenheit)
```
Comment (berlam) [inline README.md]: Credits already updated.

File: README.md [hunk 2]
```diff
@@ -16,57 +16,61 @@ Features:
...
+and later improved upon by [Mattias Berla](https://github.com/berlam).
 
 ## License ##
 
-TBD
+[ISC License](https://en.wikipedia.org/wiki/ISC_license), functionally
```
Comment (berlam) [inline README.md]: License already updated

File: LICENSE [hunk 3]
```diff
@@ -0,0 +1,16 @@
+Awesome Alt-Tab Switcher
```
Comment (berlam) [inline LICENSE]: No need for the LICENSE as LICENSE.md is now in the repository.

**Pull request closed without merging**